Add Wikidata database link for `aceview.worm`

CondeArcana333 opened the pull request.

cthoyt commented:
@CondeArcana333 please share with me your personal ORCID identifier so I can show you how to mark your contributions

CondeArcana333 commented:
> añadió
disculpe la demora, le proporciono mi codigo orcid personal 
https://orcid.org/0000-0002-5283-0175

cthoyt commented on src/bioregistry/data/bioregistry.json:
```diff
@@ -284,7 +306,10 @@
       "prefix": "ACEVIEW.WORM",
-      "uri_format": "http://identifiers.org/aceview.worm/$1"
+      "uri_format": "http://identifiers.org/aceview.worm/$1",
+      "wikidata": {
```
the wikidata key should not be inside the "prefixcommons" dictionary

CondeArcana333 commented:
good evening @cthoyt, I corrected the location of the wikidata link.
I wanted to tell you about the contributors, our intention is to upload the 1252 wikidata databases as well as add the databases that are not already added to wikidata. Reason for which it would be fair to add them as contributor_extras, for now I have uploaded them one by one to understand the conditions for my contributions to be accepted, but I will start uploading bigger changes.
If you agree with what I mentioned in my previous point, please check if I added the contributors correctly.

cthoyt commented:
# Attribution

The Bioregistry's job is not to attribute curation done on Wikidata. The contributor and contributor_extras sections are there to give credit to people who make material contributions via pull requests (or subjectively, other meaningful insight, but this is not the case here since this is a very simple edit). Typically the `contributor` slot is reserved for the person who originally submits a record, and `contributor_extras` are for people who edited it after the fact. I realize now this is confusing and will make an update. In your case, there should only be one entry in `contributor_extras` corresponding to you, and nobody else. Please make the corresponding updates.

# Fixing the Tests

I've added tests to make sure the bioregistry is properly linted - so notice that the tests are now failing on CI in GitHub. Any time you see this, you should read the logs from the tests to see what you need to fix. Here's a visual guide on how to do that in case you're not familiar:

## Step 1 - Click the "details" for the failing tests at the bottom of the pull request

<img width="1151" alt="Screen Shot 2022-03-17 at 11 55 11" src="https://user-images.githubusercontent.com/5069736/158794729-d5f79db8-9c1a-4dcb-96d6-697294b73319.png">

## Step 2 - Scroll up

GitHub actions automatically scrolls down to the bottom of the "step" that failed. Since Bioregistry used `tox` to run its tests, then the `tox` command failed, but inside is the actual `pytest` output, so you need to scroll up to see it

<img width="1292" alt="Screen Shot 2022-03-17 at 11 56 00" src="https://user-images.githubusercontent.com/5069736/158794995-44998dbb-ccbd-49b6-8576-35958cb90a76.png">

## Read the output

There may be potentially several issues, so I'd suggest working from the bottom up. I've done my best to add helpful messages to the tests that can tell you how to follow-up. If there's ever confusion, then I will be highly motivated to make improvements here.

<img width="1250" alt="Screen Shot 2022-03-17 at 11 56 21" src="https://user-images.githubusercontent.com/5069736/158795058-d028434e-209d-4779-810d-312aa68f30c3.png">

Note the issue right now is that the `bioregistry.json` file is not sorted. There's a command you can run from your shell to sort the Bioregistry and fix this.

cthoyt commented:
@CondeArcana333 please confirm that you understand what still needs to be done, otherwise I can try to explain a different way
